Documentation/policy/language.rst

```diff
@@ -19,9 +19,9 @@ can talk to each other. Layer 3 policies can be specified using two methods:
   destination endpoint is not controlled by Cilium.
 
 * `Entities based`: Entities are used to describe remote peers which can be
-   categorized without knowing their IP addresses. This includes connectivity
-   to the local host serving the endpoints or all connectivity to outside of
-   the cluster. Future versions will allow to define your own entities.
+  categorized without knowing their IP addresses. This includes connectivity
+  to the local host serving the endpoints or all connectivity to outside of
+  the cluster. Future versions will allow to define your own entities.
```

Comment:
Would it be helpful to link to a GitHub issue referring to this future work? I'm unsure whether we want to explicitly reference GH issues in documentation, so just an idea.

Reply:
We should have one either way if we intend to implement it, and it seems reasonable to me for it to be linked from docs. Then at least there is a contact point for users who read the docs and are excited about the feature.